circuits/cpp/barretenberg/py/src/estimation/msm.py

```diff
@@ -0,0 +1,21 @@
+class MSM:
+    def __init__(self, length):
+        self.length = length
+
+    def num_gates_native(self):
+        return 960 * self.length
```

Comment:
would be good to have comments that describe where these estimations come from

Reply:
FWIW I'm hopeful that a native scalar mul can be done in 10 ECCVM rows, and that all non-native scalar muls can be converted into ECCVM scalar muls.

A conservative estimate would cost 10 ECCVM rows to be the same as 200 UltraPlonk rows (will likely be a lot less but we need to measure)

Reply:
Yeah, I should do this. With times, the numbers come from measurements done in certain test (and you can see the `chrono`s for most (all?) of these in the C++ diff). But I agree it would be helpful to be more explicit. With gate counts, some come from measurements in tests, some come from asking Kesha, and some (ECCVM) come from memory from Valencia. Maybe I have the ECCVM part wrong though.